client/patch-react.js

```diff
@@ -6,6 +6,9 @@ import React from 'react'
 let patched = false
 
 export default (handleError = () => {}) => {
+  // Temporarly stop patching React to support native class syntax for React
+  if (React) return
+
```

Comment:
This is temp. fix.
My point is this PR doesn't add much value to our process. But it complicate stuff.

For an example, now we need to set browsers. We could set last 2 versions, then it doesn't include async-await support. Only Chrome(FF soon) has it.
So, we need to ask the browser from the user as well.

If we removed source-maps support, this is something useful.
But I'm not sure about removing it.